uber/templates/registration/arbitrary_charge_form.html

```diff
@@ -9,7 +9,7 @@
     </div>
 {% else %}
     <h2>Create Arbitrary Credit Card Charges</h2>
```

Comment:
``GET`` was more appropriate here, since then we can fill ou the form and then link someone to the result instead of making them fill out the form themselves.